src/squid.md.in

```diff
@@ -0,0 +1,234 @@
+
+# NAME
+
+squid - HTTP web proxy caching server
+
+# SYNOPSIS
+
+squid [-dhisrvzCFNRSVYX] [-\-foreground] [-l facility] [-f file] [-[au] port] [-k signal] [-n service] [-O command]
+
+# DESCRIPTION
+
+**squid** is a high-performance proxy caching server for web
```

Comment:
Please summarize (here, in a GitHub comment) the changes made by this PR to this file and their purpose, especially since GitHub does not show the diff for these changes.

Reply:
Done for textual changes other than "conversion to Markdown" already mentioned in the PR description.